nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2014,6 +2015,37 @@ async fn instance_reboot(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Provision an instance
```

Comment:
what's the difference between "provision" and "create"? How would we explain it to a user?